### ISSUE TITLE
[package] qt/5: Non-canonical cmake names

### ISSUE THREAD

**Issue by ruilvo** [conan-io/conan-center-index#5019]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **qt/5**

I was trying to write a recipe for Kirigami (#5015) and in [Kirigami's CMakeLists.txt](https://invent.kde.org/frameworks/kirigami/-/blob/master/CMakeLists.txt) one can read `find_package(Qt5 ...`, which is a valid way to look for Qt in the official docs for CMake usage. In the recipe I'm trying to make I used `generators = "cmake", "cmake_find_package_multi"` and then I got this error:

```
CMake Error at CMakeLists.txt:30 (find_package):
  Could not find a package configuration file provided by "Qt5" (requested
  version 5.14.0) with any of the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.
```

(yes, I know conan doesn't have 5.14.0, but that's not the point)

See how it looks for `qt5-config.cmake`? Well, the files conan offers are `qt-config.cmake`, `qt-config-version.cmake`, and `qtTargets.cmake`. 

So I guess this is something in need of fixing.

**Comment by ruilvo** [comment]
I also noticed `set_property(TARGET qt::qt` in the file `qt-config.cmake`, and the targets Qt uses are like `Qt5::Core` and etc...

**Comment by ericLemanissier** [comment]
this is blocked by https://github.com/conan-io/conan-center-index/issues/4529

**Comment by ericLemanissier** [comment]
This is fixed by #5691

**Comment by ruilvo** [comment]
Thanks @ericLemanissier. I wrote that on other bugs and forgot about mine!!! hahah 
Now we can start doing recipes for the KDE stuff!!
